cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts

```diff
@@ -85,6 +86,7 @@ describe(["@tier3"], "Applications interlinked to tags and business service", ()
 
         // Remove the BS and tags
         application.removeBusinessService();
+        cy.wait(5000);
```

Comment:
Can you please use `SEC`? this will make it more readable `cy.wait(5 * SEC)`

Reply:
Removed